TIPS.md

```diff
@@ -0,0 +1,31 @@
+Instructions for play
+
+- While some parts of the mod are scripted, many parts (such as exploring and building/placing of invaders after setup) are not
+- When playing single player you can ignore the ready token and the "ready" text at the upper right (it's used for coordination among players)
+- Do not hit the rewind/fastforward (they do not work well with scripted mods)
+- To look at the reverse side of your spirit panel press Alt+Shift (default keybinding in TTS) and mouse over the spirit panel 
+- To zoom in on a card hold the Alt key (default keybinding in TTS) and mouse over the card/object
+
+Intermediate Tips:
+- Press numpad keys (as shown in the bottom right) to spawn new pieces
+  - Make sure numlock is enabled
+  - Note that these scripting keys can be rebound in the TTS settings
+- Lock pieces (right-click and select lock)
+- Mark damage on pieces by changing the state
+  - i.e. to mark one damage on a town, right click the town and change the state to 2
+  - You can also press a number key to change the state (so press 2 to change the state to 2)
+  - Note that invaders that have extra health or deal extra damage are rendered in a different color (i.e. red for england buildings, and blue for russian explorers)
+- Defend values can also be set as a state
+- To prevent a piece from going away during time passes you should lock it (right-click and select lock)
+- You can use energy tokens instead of the counter by deleting the energy counter (right-click on the energy counter and press delete)
```

Comment:
would probably be helpful to add that to delete objects you typically need to be host or promoted